WebDevStudios/wd_s
# Hotfix/#460 slick carousel a11y fixes

Doesn't necessarily close a ticket. It does improve the situation for #460.

### DESCRIPTION ###
I've added some additional features to help SlickJS do better at accessible content:

- A pause button is added to the carousel through JavaScript after it initializes.
- Made use of `wp.a11y.speak()` to update an aria-live region with the carousel status, specifically when the pause button is used or when the arrow navigation is used.

The dot navigation didn't need any additions, since it seems to be using a tablist role to provide feedback about the amount of slides there are and what button is activated (eg. "3 of 3").

### SCREENSHOTS ###
No significant styling was added. The pause button will sit in the bottom left, below the slider, by default.

I do have a screenshot of a test carousel and the NVDA speech viewer:
![nvda](https://user-images.githubusercontent.com/1461585/57022785-28176d80-6bfe-11e9-94ea-4a29274aae0f.png)

### OTHER ###

### STEPS TO VERIFY ###
Create a page with a carousel on it and verify that the pause button appears. If you have NVDA or some other screenreader installed, you should be able to use that to test the verbal feedback by clicking/keyboarding the Previous/Next and Pause buttons.
